src/validators/validateCreateNotificationParams.ts

```diff
@@ -6,6 +6,11 @@ export const validateCreateNotificationParams = (
 ): Joi.ValidationResult => Joi.object().keys({
   notificationType: Joi.string().required(),
   toUserId: Joi.number().integer().min(1).required(),
+  context: Joi.object().keys({
+  }).pattern(
+    Joi.string().invalid('from', 'notification', 'message_type', 'toUserId', 'notificationType').regex(/(google).*|(gcm).*/i, { invert: true }).insensitive(),
+    Joi.string().invalid('from', 'notification', 'message_type', 'toUserId', 'notificationType').regex(/(google).*|(gcm).*/i, { invert: true }).insensitive(),
```

Comment:
This is restricting the allowed values:

```sh
curl -v -X POST \
    -H "Content-Type: application/json" \
    --data '{"toUserId":7,"notificationType":"test","context":{"a": "google"}}' \
    http://localhost:3001/api/notifications \
  |  jq .
```
```json
{
  "error": {
    "_original": {
      "toUserId": 7,
      "notificationType": "test",
      "context": {
        "a": "google"
      }
    },
    "details": [
      {
        "message": "\"context.a\" with value \"google\" matches the inverted pattern: /(google).*|(gcm).*/i",
        "path": [
          "context",
          "a"
        ],
        "type": "string.pattern.invert.base",
        "context": {
          "regex": {},
          "value": "google",
          "label": "context.a",
          "key": "a"
        }
      }
    ]
  }
}
```

I think all we need to do is require a string:

```suggestion
    Joi.string(),
```